pennylane/qchem/structure.py

```diff
@@ -412,3 +413,136 @@ def excitations_to_wires(singles, doubles, wires=None):
         doubles_wires.append([d1_wires, d2_wires])
 
     return singles_wires, doubles_wires
+
+
+def pubchem_data(identifier, identifier_type="name"):
+    r"""Obtain name, geometry and charge of a compound from the PubChem Database.
+
+    The `PubChem <https://pubchem.ncbi.nlm.nih.gov>`__ is one of the largest public repositories for
+    the information on chemical substances from which name, geometry (in Bohr radius), and charge can be
+    retrieved for a compound by its name, SMILES, InChI, InChIKey, and PubChem Compound ID (CID) to
+    build a molecule object for Hartree-Fock calculations.
+
+    Args:
+        identifier (str or int): compound's identifier as required by the PubChem database
+        identifier_type (str): type of the provided identifier - name, CAS, CID, SMILES, InChI, InChIKey
+
+    Returns:
+        Tuple(list[str], array[float], int): symbols, geometry (in Bohr radius) and charge of the compound
+
+    **Example**
+
+    >>> mol_data_from_pubchem("BeH2")
```

Comment:
Looking at this example, here's what I'd like to see as a user:
1. Just `mol_data`. We can specify in the docstring that it comes from pubchem
2. `symbols, coordinates = mol_data('H2O')`
3. That `symbols, coordinates` are in the right format for pennylane
4. Make sure that the units are correct